portafly/portafly_api.yml

```diff
@@ -7,35 +7,55 @@ info:
     A tuned API for PortaFly, a Patternfly SPA consuming 3scale Porta API MGMT solution.
   license:
     name: Apache 2.0
-    url: https://www.apache.org/licenses/LICENSE-2.0
+    url: "https://www.apache.org/licenses/LICENSE-2.0"
 paths:
   /developer/accounts:
     summary: A list of developer accounts
     description: A list of developer accounts.
     get:
       responses:
         "200":
-          $ref: '#/components/responses/developerAccountList'
+          $ref: "#/components/responses/developerAccountList"
       operationId: developerAccountList
       summary: Get developer account list
     parameters:
-    - name: page
-      description: Page in the paginated list. Defaults to 1.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: per_page
-      description: Number of results per page. Default and max is 500.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: state
-      description: Account state.
-      schema:
-        type: string
-      in: query
```

Comment:
What did it change here? and what triggered that change?

Reply:
This is in fact the correct format for arrays, with 1 indentation:
```
array:
  - item1
  - item2
```

Reply:
Was this triggered by Apicurio online, @damianpm ? remember to put them in a separated commit

Reply:
Apicurio online apply it's own formatting, but it's different:
```
parameters:
            -
                name: page
                description: Page in the paginated list. Defaults to 1.
                schema:
                    type: integer
                in: query
                required: false
            -
                name: per_page
                description: Number of results per page. Default and max is 500.
                schema:
                    type: integer
                in: query
                required: false
            -
                name: state
                description: Account state.
                schema:
                    type: string
                in: query
```

Reply:
I moved this to a new commit @didierofrivia